template/en/default/attachment/create.html.tmpl

```diff
@@ -84,7 +84,7 @@ TUI_hide_default('attachment_text_field');
         <td>
           <em>If you want to assign this [% terms.bug %] to yourself,
               check the box below.</em><br>
```

Comment:
correction: if we're going to pre-check the checkbox, the text should be updated to reflect that probably.